emperor/support_files/js/shape-controller.js

```diff
@@ -0,0 +1,121 @@
+define([
+    "jquery",
+    "underscore",
+    "view",
+    "viewcontroller",
+    "three",
+    "shape-editor",
+    "shapes"
+], function ($, _, DecompositionView, ViewControllers, three, Shape, shapes) {
+
+  // we only use the base attribute class, no need to get the base class
+  var EmperorAttributeABC = ViewControllers.EmperorAttributeABC;
+  var ShapeEditor = Shape.ShapeEditor;
+  var ShapeFormatter = Shape.ShapeFormatter;
+  /**
+   * @name ShapeController
+   *
+   * @class Manipulates and displays the shape of objects on screen.
+   * @property {String} [title=""] Title of the controller.
```

Comment:
Can you remove all the `@property` elements? And replace them with `@inherits EmperorAttributeABC`?